trie/triedb/pathdb/database.go

```diff
@@ -40,11 +40,11 @@ const (
 	// defaultCleanSize is the default memory allowance of clean cache.
 	defaultCleanSize = 16 * 1024 * 1024
 
-	// maxBufferSize is the maximum memory allowance of node buffer.
+	// MaxBufferSize is the maximum memory allowance of node buffer.
 	// Too large nodebuffer will cause the system to pause for a long
 	// time when write happens. Also, the largest batch that pebble can
 	// support is 4GB, node will panic if batch size exceeds this limit.
-	maxBufferSize = 256 * 1024 * 1024
+	MaxBufferSize = 256 * 1024 * 1024
```

Comment:
Nitpick. maxBufferSize was kind of ok as a name for a private field, but being public, it's not descriptive enough. Suggest `MaxDirtyBufferSize`